apps/qubit/modules/actor/templates/browseSuccess.php

```diff
@@ -1,103 +1,172 @@
-        'name' => 'subjects',
-        'aggs' => $aggs,
-        'filters' => $search->filters)) ?>
+      <div class="visible-phone facets-header">
+        <a class="x-btn btn-wide">
+          <i class="fa fa-filter"></i>
+          <?php echo __('Filters') ?>
+        </a>
+      </div>
 
-    </div>
+      <div class="content">
+
+        <h2><?php echo sfConfig::get('app_ui_label_facetstitle') ?></h2>
+
+        <?php echo get_partial('search/aggregation', array(
+          'id' => '#facet-languages',
+          'label' => __('Language'),
+          'name' => 'languages',
+          'aggs' => $aggs,
+          'filters' => $search->filters)) ?>
+
+        <?php echo get_partial('search/aggregation', array(
+          'id' => '#facet-entitytype',
+          'label' => __('Entity type'),
+          'name' => 'entityType',
+          'aggs' => $aggs,
+          'filters' => $search->filters)) ?>
+
+        <?php echo get_partial('search/aggregation', array(
+          'id' => '#facet-maintainingrepository',
+          'label' => __('Maintained by'),
+          'name' => 'repository',
+          'aggs' => $aggs,
+          'filters' => $search->filters)) ?>
+
+        <?php echo get_partial('search/aggregation', array(
+          'id' => '#facet-occupation',
+          'label' => __('Occupation'),
+          'name' => 'occupation',
+          'aggs' => $aggs,
+          'filters' => $search->filters)) ?>
+
+        <?php echo get_partial('search/aggregation', array(
+          'id' => '#facet-places',
+          'label' => sfConfig::get('app_ui_label_place'),
+          'name' => 'places',
+          'aggs' => $aggs,
+          'filters' => $search->filters)) ?>
+
+        <?php echo get_partial('search/aggregation', array(
+          'id' => '#facet-subjects',
+          'label' => sfConfig::get('app_ui_label_subject'),
+          'name' => 'subjects',
+          'aggs' => $aggs,
+          'filters' => $search->filters)) ?>
 
-  </section>
-<?php end_slot() ?>
+      </div>
+
+    </section>
+  <?php end_slot() ?>
+
+<?php endif; ?>
 
 <?php slot('before-content') ?>
 
   <section class="header-options">
+    <?php if (isset($sf_request->hasDigitalObject)): ?>
+      <span class="search-filter">
+        <?php if (filter_var($sf_request->hasDigitalObject, FILTER_VALIDATE_BOOLEAN)): ?>
+          <?php echo __('With digital objects') ?>
+        <?php else: ?>
+          <?php echo __('Without digital objects') ?>
+        <?php endif; ?>
+        <?php $params = $sf_data->getRaw('sf_request')->getGetParameters() ?>
+        <?php unset($params['hasDigitalObject']) ?>
+        <a href="<?php echo url_for(array('module' => 'actor', 'action' => 'browse') + $params) ?>" class="remove-filter"><i class="fa fa-times"></i></a>
+      </span>
+    <?php endif; ?>
+
+    <?php if (isset($repository)): ?>
+      <?php echo get_component('search', 'filter', array('object' => $repository, 'param' => 'repository')) ?>
+    <?php endif; ?>
```

Comment:
Nice component!

To make it even better we could move the if to the component itself, to avoid adding one for each component call. The check can be added at the beginning of the execute function in the component and return `sfView::NONE` directly.

It would be great to use it in the IO browse page where possible too, but we can do that later if you think it's out of scope.

Reply:
Nice... I'll try that. I'll make an issues to incorporate this cimponent into the IO browse page.